gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/token/createAccount.ts

```diff
@@ -0,0 +1,56 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { ASSOCIATED_TOKEN_PROGRAM_ID, Token, TOKEN_PROGRAM_ID } from '@solana/spl-token'
+import { PublicKey } from '@solana/web3.js'
+import { CONTRACT_LIST } from '../../../lib/contracts'
+
+export default class CreateAccount extends SolanaCommand {
+  static id = 'token:create_account'
+  static category = CONTRACT_LIST.TOKEN
+  static examples = ['yarn gauntlet token:create_account --network=devnet --address=<BASE_ADDRESS> <TOKEN>']
```

Comment:
I think we need to start adding `description/overview/summary` for every command to help users understand what's going on in more detail. Keep in mind for the future.